docs/server/features/connectors/features.md

```diff
@@ -275,4 +275,35 @@ For more details on customizing these settings, refer to the [Resilience Configu
 
 ::: note
 Some connectors have their own resilience mechanisms and configurations. Refer to the specific connector's page for details.
+:::
+
+## Data Protection
+
+KurrentDB Connectors implement automatic data protection for sensitive
+configuration fields. This security feature ensures that confidential
+information, such as passwords and access tokens, is encrypted during
+transmission.
+
+When you configure a connector, certain sensitive fields are automatically encrypted:
+
+- The system automatically identifies sensitive keys in the configuration using predefined patterns determined internally
+- These values are encrypted using a token provided by you
+- The encrypted values are stored securely in KurrentDB instead of plaintext
+- This protection happens automatically without user configuration
```

Comment:
```suggestion
When you configure a connector, the system identifies sensitive keys using predefined internal patterns. It encrypts these keys with a token that you provide. The encrypted data is stored securely in KurrentDB rather than in plaintext. This protection is applied automatically without additional configuration.
```